src/django/pfb_analysis/functions.py

```diff
@@ -0,0 +1,14 @@
+from django.db.models.expressions import Func
+
+
+class ObjectAtPath(Func):
```

Comment:
I have some hunches on why that F expression is failing, but either way I wouldn't expect them to work for jsonb fields since the SQL query syntax is different and jsonb is postgresql specific. Either way I think the solution presented here is ok for now.